README.md

```diff
@@ -18,6 +18,11 @@ Tools for working with tabular data in Julia.
 **Maintenance**: DataTables is maintained collectively by the [JuliaData collaborators](https://github.com/orgs/JuliaData/people).
 Responsiveness to pull requests and issues can vary, depending on the availability of key collaborators.
 
+For reading and writing tabular data, load the [CSV.jl](https://github.com/JuliaData/CSV.jl) package with
```

Comment:
I'd rather put this in the manual. Else CSV would be the only (non-)feature mentioned on the home page...